flask/main.py

```diff
@@ -118,6 +132,19 @@ def unhandled_exception():
     obj = {}
     obj['keyDoesntExist']
 
+@app.route('/api', methods=['GET'])
```

Comment:
No sleep timeouts yet. Let's make sure all works as expected and we comprehend it.